okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/RemotingOkHttpCall.java

```diff
@@ -269,6 +278,10 @@ public void onResponse(Call call, Response response) throws IOException {
                 callback.onFailure(call, new SafeIoException("Failed to handle request, "
                         + "this is an conjure-java-runtime bug."));
             }
+
+            private boolean isStreamingBody(Call call) {
+                return call.request().body() instanceof UnrepeatableRequestBody;
```

Comment:
A bit worried about relying on an internal API here, which is gone from okhttp 3.14, though I'm also not sure if they intend to continue development on the 3.x branch.
Do you think we should try to use reflection here to be defensive?

Reply:
My guess is we're not sure we'll be upgrading any time soon and once you do this will just fail to compile. If we upgrade there is a public API for checking this.

Can you suggest how would you solve this with reflection?